irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -62,7 +64,8 @@ namespace iroha {
         transaction_.exec(
             "INSERT INTO role_has_permissions(role_id, permission_id) VALUES "
             + std::accumulate(
-                  std::next(permissions.begin()), permissions.end(),
+                  std::next(permissions.begin()),
+                  permissions.end(),
                   entry(*permissions.begin()),
                   [&entry](auto acc, auto x) { return acc + ", " + entry(x); })
```

Comment:
this is not even 1st normal form :(

Reply:
it isn't. It inserts multiple rows at once. Not multiple values into single row